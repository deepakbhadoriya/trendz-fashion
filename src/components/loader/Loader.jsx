import React from 'react';

import Logo from '../../assets/icons/logo/logo';

const Loader = () => {
  return (
    <center>
      <Logo />
    </center>
  );
};

export default Loader;
