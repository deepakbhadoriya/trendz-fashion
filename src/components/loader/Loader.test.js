import React from 'react';

import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import Loader from './Loader';

const { mount } = Enzyme;
Enzyme.configure({ adapter: new Adapter() });

describe('<Loader/>', () => {
  it('render correctly (snapshot)', () => {
    const tree = mount(<Loader />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
