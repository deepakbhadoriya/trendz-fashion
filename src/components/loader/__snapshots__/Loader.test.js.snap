// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> render correctly (snapshot) 1`] = `
<Loader>
  <center>
    Loader
  </center>
</Loader>
`;
